easybuild/easyconfigs/p/PaStiX/PaStiX-6.0.0-intel-2018a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'PaStiX'
+version = '6.0.0'
+
+homepage = 'http://pastix.gforge.inria.fr/'
+description = """PaStiX (Parallel Sparse matriX package) is a scientific library that provides a high performance
+ parallel solver for very large sparse linear systems based on direct methods."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://gforge.inria.fr/frs/download.php/file/37244/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = [
+    '1dc37475964d7baa0df30c5909bfdf459dd0db66b866f9d9f566ab6fd7e12d7f',  # pastix-6.0.0.tar.gz
+    '2b898a2bc8c5e97272fa61c8365f6332222813fc02ea4188cbad089e72f50e92',  # pastix-intel.patch
+]
+
+patches = ['pastix-intel.patch']
```

Comment:
Please move up above `checksums`, and rename the patch file to something like `PaStiX-6.0.0_intel.patch`